src/content/ci/github-actions.mdx

```diff
@@ -496,11 +496,13 @@ Read the official [Storybook environment variable's documentation](https://story
 
 ### Recommended configuration for build events
 
-GitHub's Actions, like other CI systems, can run based on any type of event. Our recommendation is to run Chromatic's step on `push` events. All other event types except `pull-request` will not work.
+GitHub's Actions, like other CI systems, can run based on any type of event. Our recommendation is to run Chromatic's step on `push` events.
 
-The `pull-request` event requires special consideration. Like other CI systems, GitHub allows workflow execution on either commits pushed to a branch in a pull request. Or for "merge" commits between that branch and the base branch (main).
+All other event types except `pull_request` will not work. However, using the `pull_request` event might cause Chromatic's baselines to be lost in certain scenarios.
```

Comment:
> All other event types except `pull_request` will not work.

Not quite sure what this is supposed to say, but this is not true.

Reply:
It's old wording but, I think it's meant to say that only `pull_request` and `push` events work. I should reword it but, also, If that's not true then we can drop it.

Reply:
> using the `pull_request` event might cause Chromatic's baselines to be lost in certain scenarios

The other thing that can happen is that unexpected baselines from the `main` branch are found, which is explained below, but maybe needs to be added here too so this doesn't sound like it's all about losing baselines?

Reply:
> It's old wording but, I think it's meant to say that only `pull_request` and `push` events work. I should reword it but, also, If that's not true then we can drop it.

Ah! Ok yeah I get that from reading the old version, but that's less clear on the updated version.

Reply:
I had to check, it looks like we do support a handful of events:

`pull_request`, `pull_request_review`, `pull_request_target`, `push`, `workflow_run`, `workflow_dispatch`, `issue_comment`, `schedule`, `release`, `merge_group`.

(From here: https://github.com/chromaui/chromatic-cli/blob/main/action-src/main.ts#L24-L82)

I wonder how many of those need a workaround like `pull_request` 🤔

Reply:
Do we want to mention them all?

Reply:
As discussed in slack, we'll revisit this during the cool down cycle